README.md

```diff
@@ -1,6 +1,6 @@
 # ghost-advisory-theme
 
-Custom-built Ghost theme for [Advisory SG](https://advisory.sg).
+Custom-built Ghost theme for [Advisory SG](https://advisory.sg), based on the Starter theme.
```

Comment:
Could you add this link to "Starter theme", https://github.com/TryGhost/Starter, in Markdown format?